src/web/src/pages/search.tsx

```diff
@@ -1,11 +1,14 @@
 import SEO from '../components/SEO';
 import SearchPage from '../components/SearchPage';
+import SearchProvider from '../components/SearchProvider';
 
 const Search = () => {
   return (
     <>
-      <SEO pageTitle="Search | Telescope" />
-      <SearchPage />
+      <SearchProvider>
```

Comment:
I am thinking if it is still a good practice to keep things wrapped in react fragment here or should we just remove it since `SearchProvider` wraps everything up? 🤔

Reply:
It's a minor thing, works pretty much the same with or without it

Reply:
Attention to details is key 😉